src/frontend/screens/Settings/ProjectSettings/index.tsx

```diff
@@ -105,6 +126,19 @@ export const ProjectSettings = () => {
           onPress={() => navigate('YourTeam')}
         />
       )}
+      {isCoordinator && (
+        <SettingsCardRow
+          icon={<ExchangeIcon width={24} height={24} />}
+          title={formatMessage(
+            remoteArchiveOn ? m.remoteArchiveOn : m.remoteArchiveOff,
+          )}
+          subtitle={formatMessage(m.remoteArchiveDesc)}
+          buttonText={formatMessage(m.viewDetails)}
+          onPress={() =>
+            navigate(remoteArchiveOn ? 'RemoteArchiveOn' : 'RemoteArchiveOff')
+          }
+        />
+      )}
```

Comment:
this doesn't properly guard against the issue i ran into. needs an additional check like `isCoordinator && projectInfo.name !== undefined`

Reply:
unless the intention is to be okay with running into the error in that case? i'd be okay with that too but need some clarity on what the intended UX is